src/toxcore/dht/server/mod.rs

```diff
@@ -616,7 +616,8 @@ impl Server {
             (packet, addr)
         }).collect::<Vec<_>>();
 
-        send_all_to_bounded(&self.tx, stream::iter_ok(packets), Duration::from_secs(DHT_SEND_TIMEOUT))
+        Box::new(send_all_to_bounded(&self.tx, stream::iter_ok(packets), Duration::from_secs(DHT_SEND_TIMEOUT))
```

Comment:
Why Box?

Reply:
https://github.com/tox-rs/tox/blob/65174889f4301fcca24da059fd436b9f5558ee59/src/toxcore/dht/server/mod.rs#L603
Because return type of function is still IoFuture<()>. It is temporary used. After we change all IoFuture<()> to impl Future, this Box can be removed.